lit/docs/jobs/steps.lit

```diff
@@ -1037,6 +1040,148 @@ by configuring \reference{schema.step.on_failure} or
     }
   }
 
+  \optional-attribute{across}{[across_var]}{
+    Run a wrapped step multiple times with different combinations of variable
```

Comment:
I know in the code the substep is wrapped, but in the pipeline yaml where you write `across` it's at the same level as the step it's manipulating.
```suggestion
    Run a step multiple times with different combinations of variable
```